Workbooks/Intune Devices/Devices/IntuneDevices.workbook

```diff
@@ -0,0 +1,270 @@
+              "formatter": 3,
+              "formatOptions": {
+                "showIcon": true
+              }
+            }
+          ]
+        }
+      },
+      "customWidth": "50",
+      "name": "Query - DevicesByComplianceState",
+      "styleSettings": {
+        "maxWidth": "50"
+      }
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "IntuneDevices\r\n| where TimeGenerated > ago(1d)\r\n| where Result != \"Completed\"\r\n| join kind = inner (\r\n    IntuneDevices\r\n    | where TimeGenerated > ago(1d)\r\n    | where Result == \"Completed\"\r\n    | summarize arg_max(TimeGenerated, BatchId)\r\n) on BatchId\r\n| summarize arg_max(DeviceId, *) by DeviceId\r\n| summarize count() by ManagedBy",
+        "size": 3,
+        "title": "Devices By Management Agent",
+        "showExportToExcel": true,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "piechart",
+        "gridSettings": {
+          "formatters": [
+            {
+              "columnMatch": "OperationCount",
+              "formatter": 3,
+              "formatOptions": {
+                "showIcon": true
+              }
+            }
+          ]
+        }
+      },
+      "customWidth": "50",
+      "name": "Query - DevicesByManagedBy",
+      "styleSettings": {
+        "maxWidth": "50"
+      }
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "IntuneDevices\r\n| where TimeGenerated > ago(1d)\r\n| where Result != \"Completed\"\r\n| where OS has \"Android\"\r\n| join kind = inner (\r\n    IntuneDevices\r\n    | where TimeGenerated > ago(1d)\r\n    | where Result == \"Completed\"\r\n    | summarize arg_max(TimeGenerated, BatchId)\r\n) on BatchId\r\n| summarize arg_max(DeviceId, *) by DeviceId\r\n| summarize DeviceCount = count() by OSVersion\r\n| sort by OSVersion desc\r\n| project AndroidVersion = OSVersion, DeviceCount",
+        "size": 0,
+        "title": "Android Device Count By Version",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "visualization": "table"
+      },
+      "customWidth": "50",
+      "name": "Query - AndroidDeviceCountByVersion",
+      "styleSettings": {
+        "maxWidth": "50"
+      }
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "IntuneDevices\r\n| where TimeGenerated > ago(1d)\r\n| where Result != \"Completed\"\r\n| where OS has \"iOS\"\r\n| join kind = inner (\r\n    IntuneDevices\r\n    | where TimeGenerated > ago(1d)\r\n    | where Result == \"Completed\"\r\n    | summarize arg_max(TimeGenerated, BatchId)\r\n) on BatchId\r\n| summarize arg_max(DeviceId, *) by DeviceId\r\n| summarize DeviceCount = count() by OSVersion\r\n| sort by OSVersion desc\r\n| project AndroidVersion = OSVersion, DeviceCount",
```

Comment:
FYI, all these other types have "AndroidVersion" as the column name, even for iOS and windows

Reply:
Good catch! That is not correct. I will fix it.

Reply:
Fixed!